File: wear/src/main/java/info/nightscout/androidaps/complications/WallpaperDarkComplication.java
package info.nightscout.androidaps.complications;

/*
 * Created by dlvoy on 2019-11-12
 */
public class WallpaperDarkComplication extends WallpaperComplication {

    @Override
    public String getWallpaperAssetsFileName() {
        return "watch_dark.jpg";
    }

    @Override
    public String getProviderCanonicalName() {
        return WallpaperDarkComplication.class.getCanonicalName();
    }

    @Override
    public ComplicationAction getComplicationAction() {
        return ComplicationAction.NONE;
    };
}

File: wear/src/main/kotlin/app/aaps/wear/complications/WallpaperDarkComplication.kt
package app.aaps.wear.complications

/**
 * Dark Wallpaper Complication
 *
 * Displays dark-themed wallpaper image
 * Asset: watch_dark.jpg
 *
 */
class WallpaperDarkComplication : WallpaperComplication() {

    override val wallpaperAssetsFileName: String = "watch_dark.jpg"
    override fun getProviderCanonicalName(): String = WallpaperDarkComplication::class.java.canonicalName!!
}